src/Apps/Order/Components/PollAccountBalance.tsx

```diff
@@ -62,16 +68,22 @@ const PollAccountBalance: FC<PollAccountBalanceProps> = ({
       balanceCents >= buyerTotalCents && currencyCode === orderCurrencyCode
     )
 
-    onBalanceCheckComplete(!enoughBalance)
+    onBalanceCheckComplete(
```

Comment:
Here, reading the code, i'm like "ok, this is external, what is this 👀 really doing 👀. It's importance is immediately elevated at a glance.